queries.css

```diff
@@ -168,4 +212,37 @@
     justify-content: space-between;
     align-items: center;
   }
+
+  .categories-img {
+    flex: 1 1 400px;
+    background-size: cover;
+  }
+
+  .categories-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: repeat(3, 1fr);
+  }
+
+  .ln-grid {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
```

Comment:
## Thumbs up 👍 
✅ Great job showcasing the greatness that comes with arranging items on the UI using the `grid` box rule. I love how you pulled this off in a neat and minimalistic way. Awesome stuff.